src/DynamoCoreWpf/Controls/StartPage.xaml.cs

```diff
@@ -159,6 +163,12 @@ internal StartPageViewModel(DynamoViewModel dynamoViewModel)
 
             #region Reference List
 
+            references.Add(new StartPageListItem(Resources.StartPageWhatsNew, "icon-discussion.png")
+            {
+                ContextData = ButtonNames.ShowGallery,
+                ClickAction = StartPageListItem.Action.RegularCommand
+            });
```

Comment:
@Benglin, as I didn't have the icon for the what's new item, I used the same one with the icon-discussion one. It will be replaced later